zanata-client-commands/src/main/java/org/zanata/client/commands/push/RawPushCommand.java

```diff
@@ -241,6 +423,25 @@ public void visit(LocaleMapping locale,
     }
 
     /**
+     * Search and return file type with given source file extension
+     *
+     * @param fileTypes
+     * @param fileExtension
+     * @return
+     */
+    private @Nullable DocumentType getFileType(Map<DocumentType, Set<String>> fileTypes,
```

Comment:
If user customised the extensions, then it will need the `Map<DocumentType, Set<String>>`  and makes more sense to be in this class.